src/components/PostDetails.tsx

```diff
@@ -1,106 +1,108 @@
 import React from 'react';
-import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
+import { Post } from '../types/Post';
+import { Comment } from '../types/Comment';
+import { Loader } from './Loader';
+
+type Props = {
+  post: Post;
+  isLoadingComments: boolean;
+  error: string | null;
+  comments: Comment[];
+  formOpen: (marker: boolean) => void;
+  isFormOpen: boolean;
+  deleteComment: (id: number) => void;
+  addComment: (comment: Comment) => void;
+  commentLoading: () => void;
```

Comment:
The `commentLoading` prop is defined as a function with no parameters, but it should accept a boolean parameter to indicate the loading state. Update the type definition to `commentLoading: (marker: boolean) => void;` to match its usage in the `NewCommentForm` component.